src/main/java/piano/SaveAudio/SaveOption.java

```diff
@@ -0,0 +1,23 @@
+package piano.SaveAudio;
+
+import piano.recorder.KeyPressedInfo;
+
+import javax.swing.*;
+import java.util.ArrayList;
+
+public class SaveOption extends JFrame {
```

Comment:
Why is this a frame?

Reply:
I thought that to have a JOptionPane you would need to extend JFrame.
Instead of the JOptionPane taking in the paremeter of this (which requires for the file to extend JFrame), should it take in the PianoGUI? Or something completely different?

Reply:
You can pass the JFrame you are currently in to JOptionPane, you don't need a new JFrame.

Reply:
How would I do that? Should I pass the frame through the different classes to get it to the SaveOption? 
Or can I extend Component instead of JFrame?

Reply:
pass the frame